src/plugins/WebIndexPlugin.ts

```diff
@@ -48,7 +49,7 @@ $bundles
         }
 
         let jsTags = bundlePaths.map(bundle =>
-            `<script type="text/javascript" src="${bundle}"></script>`
+            `<script ${opts.async ? 'async' : ''} type="text/javascript" src="${bundle}"></script>`
```

Comment:
Sorry if that sounds half assed, It's my first open source contribution and I assumed it's just a minor tweak